src/user-meta/framework/custom-meta/content-analysis-disable.php

```diff
@@ -36,4 +71,37 @@ public function get_field_id(): string {
 	public function is_empty_allowed(): bool {
 		return true;
 	}
+
+	/**
+	 * Renders the custom meta's field in the user form.
+	 *
+	 * @param int $user_id The user ID.
+	 *
+	 * @return void
+	 */
+	public function render_field( $user_id ): void {
+		echo '
+
+		<input
+			class="yoast-settings__checkbox double"
+			type="checkbox"
+			id="' . \esc_attr( $this->get_field_id() ) . '"
+			name="' . \esc_attr( $this->get_field_id() ) . '"
+			aria-describedby="wpseo_content_analysis_disable_desc"
```

Comment:
Might be nice to use the field ID as prefix?
Same for where added of course.

Reply:
You mean as in `$this->get_field_id() . '_desc', right? Makes sense, will do

Reply:
[Done](https://github.com/Yoast/wordpress-seo/pull/21314/commits/c6663452895273cf662040a21ea60dbf48e57605)